job-runner.toml

```diff
@@ -0,0 +1,56 @@
+# Default configuration for job-runner
+
+# This file contains the default settings for job-runner and will always be
+# read in order to get the defaults.  It is also meant to act as a rudimentary
+# form of documentation for the options which are available.
+
+# Anything in this file can be overridden by installing a file with a similar
+# format in one of the following locations (in order of precedence):
+#   - the path specified via `--config-file`
+#   - the path specified via the `JOB_RUNNER_CONFIG` environment variable
+#   - XDG_CONFIG_HOME/cockpit-dev/job-runner.toml
+
+# The defaults from this file are merged with overrides from the first such
+# file found.
+
+# The default configuration is intentionally broken.  You'll need to provide a
+# configuration which (at least) does one of:
+#  - provides a valid GitHub API token; and/or
+#  - sets forge.github.post to false
+
+[container]
+command = ['podman']
+run-args = [
+  # '--device=/dev/kvm'
+]
+default-image = 'quay.io/cockpit/tasks:latest'
+
+[container.secrets]
+# see podman-secret(1)
+# github-token = ['--secret=github-token']
+
+[logs]
+driver='local'   # 's3' or 'local'
+
+[logs.s3]
+# hint: podman run --rm --net=host quay.io/minio/minio server /var
+url = 'http://127.0.0.1:9000/tmp/'
```

Comment:
This is a bit awkward -- production has to duplicate `LOG_STORE` from lib/stores.py. Commenting this out and defaulting to `LOG_STORE` would be nicer.

Reply:
But nevermind, that'd go against the "become independent from bots" idea.